### PR TITLE
[quest] ADDED: 'Elixir of Insight' Quest To QuestDB

### LINKED ISSUE #5371: [quest]: 'Elixir of Insight' Missing From QuestDB
## Description

The SoD quest 'Elixir of Insight' is not currently in the QuestDB. 

## Wowhead link

https://www.wowhead.com/classic/quest=78561/elixir-of-insight

### Discord Bug Report

Quest ID: 78561
Quest Name: Elixir of Insight
Quest Text: The elixir is ready, <playername>. Go down to the bonfire, on the beach, and drink it. Heed the vision given to you by the spirits, and return to me with your findings.
Objective Text: Drink the Elixir of Insight near the bonfire at Zoram'gar Outpost, and witness the vision it grants you.
Character: Lvl 25 ORC SHAMAN
Player Coords:  [1440]  (11.424, 33.137)
QuestLog: 78922, 6462, 6563, 509, 1131, 1195, 1065, 843, 1058, 1088, 868, 541, 78561, 6381, 5881, 6962, 
Client: 1.15.0 SoD
Questie: v9.2.10
Locale: enUS

## Screenshots

N/a

## Questie & WoW version

v9.2.9
WoW Classic SoD - v1.15.052409

### PR DESCRIPTION
**Pull request by Sommos**
## Issue references

Fixes #5371

## Proposed changes

- ADDED:  'Elixir of Insight' is now in the QuestDB, and should be accessible to users. This is the second quest of the same name, but involves the player using a phial on the beach, and as such has been placed as part 3 for this shaman questline.

### CODE REVIEW

File: Database/Corrections/sodQuestFixes.lua [hunk 1]
```diff
@@ -296,6 +296,10 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.preQuestSingle] = {78506},
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [78561] = {
+            [questKeys.preQuestSingle] = {78537},
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
         [78575] = {
             [questKeys.objectives] = {{{212694}}},
             [questKeys.preQuestSingle] = {78537},
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:305]: Part 4 needs the new Part 3 as pre quest:

```suggestion
            [questKeys.preQuestSingle] = {78561},
```

File: Database/Corrections/sodQuestFixes.lua [hunk 2]
```diff
@@ -296,6 +296,10 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.preQuestSingle] = {78506},
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [78561] = {
+            [questKeys.preQuestSingle] = {78537},
+            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
+        },
```
Comment (BreakBB) [inline Database/Corrections/sodQuestFixes.lua:302]: The scraper does not handle "Provided Item:" from wowhead yet, so we manually need to remove this objective.

```suggestion
        [78561] = {
            [questKeys.objectives] = {},
            [questKeys.preQuestSingle] = {78537},
            [questKeys.zoneOrSort] = sortKeys.SHAMAN,
        },
```

File: Database/Corrections/sodQuestFixes.lua (hunk 1 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Slipped my mind! 

Fixed in: https://github.com/Questie/Questie/pull/5372/commits/50aa1f1e4414da0832b6e2061f3726f0374ac68a

File: Database/Corrections/sodQuestFixes.lua (hunk 2 above)
Comment (Sommos) [reply Database/Corrections/sodQuestFixes.lua]: Okay. I'll have to bear that in mind in the future :)

Fixed in: https://github.com/Questie/Questie/pull/5372/commits/bdea28a71cfdbc7f54879121a7c3f040a2a46326

**Pull request merged**